engine/src/test/scala/cromwell/engine/workflow/tokens/JobTokenDispenserActorSpec.scala

```diff
@@ -337,20 +337,22 @@ class JobTokenDispenserActorSpec extends TestKitSuite
         jobExecutionTokenDispenserActorName = "skip-dead-repeatedly",
       )
     val grabberSupervisor = TestActorRef(new StoppingSupervisor(), "skip-dead-repeatedly-supervisor")
-    // The first 5 get a token and the 6th and 7h one are queued
+    // The first 5 get a token and the 6th and 7th one are queued
     val tokenGrabbingActors = (0 until 1000).toVector.map { i =>
       TestActorRef[TestTokenGrabbingActor](TestTokenGrabbingActor.props(actorRefUnderTest, LimitedTo5Tokens), grabberSupervisor, s"grabber_" + i)
     }
 
-    val actorIterator = tokenGrabbingActors.toIterator
+    // Create a sliding window of 10 actors, skipping by 10 so the windows do not overlap.
+    val actorIterator = tokenGrabbingActors.sliding(10, 10)
```

Comment:
The baseline code was doing some "interesting" things that don't seem to jibe with the `Iterator` contract and consistently fail with 2.13.